lib/index.js

```diff
@@ -331,22 +341,31 @@ function _validateReceiveHeader(header, value) {
       header + ' was not in the request', 'SyntaxError');
   }
 }
+function addValue({result, validate, key, values, label}) {
```

Comment:
Make sure there is whitespace above the function.

Reply:
Addressed in https://github.com/digitalbazaar/http-signature-header/pull/12/commits/c3db9340863113950472faa881437a6dcc154eaa